contracts/HATVaultsRegistry.sol

```diff
@@ -264,52 +264,26 @@ contract HATVaultsRegistry is IHATVaultsRegistry, Ownable {
     }
 
     /** @notice See {IHATVaultsRegistry-createVault}. */
-    function createVault(
-        IERC20 _asset,
-        address _owner,
-        address _committee,
-        IRewardController _rewardController,
-        uint16 _maxBounty,
-        HATVault.BountySplit calldata _bountySplit,
-        string calldata _descriptionHash,
-        uint32 _bountyVestingDuration,
-        uint32 _bountyVestingPeriods,
-        bool _isPaused
-    ) 
-    external 
-    returns(address vault)
-    {
+    function createVault(IHATVault.VaultInitParams calldata _params) external returns(address vault) {
```

Comment:
actually i liked the separate arguments better though :)

Reply:
Yeah but get stack too deep